### ISSUE TITLE
Getting Started Error

### ISSUE THREAD

**Issue by Cacivy** [parcel-bundler/parcel#72]
`npm install -g parcel-bundler`

OS: windows 10

```
λ parcel index.html
Server running at http://localhost:1234
⏳  Building index.html...
(node:9316) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): R
angeError: Maximum call stack size exceeded
(node:9316) DeprecationWarning: Unhandled promise rejections are deprecated. In the future, pr
omise rejections that are not handled will terminate the Node.js process with a non-zero exit
code.
(node:9316) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): R
angeError: Maximum call stack size exceeded
(node:9316) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 3): R
angeError: Maximum call stack size exceeded
(node:9316) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 4): R
angeError: Maximum call stack size exceeded
(node:9316) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 5): R
angeError: Maximum call stack size exceeded
(node:9316) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 6): R
angeError: Maximum call stack size exceeded
(node:9316) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 7): R
angeError: Maximum call stack size exceeded
(node:9316) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1268)
: RangeError: Maximum call stack size exceeded
```

**Comment by hearsay316** [comment]
我也遇到了,说是在版本48里解决了. https://github.com/parcel-bundler/parcel/pull/48  .你可以等等看.

**Comment by devongovett** [comment]
Released a fix in v1.0.3.
